mackup/applications/robo3t.cfg

```
@@ -0,0 +1,5 @@
+[application]
+name = Robo 3T
+
+[configuration_files]
+Library/Preferences/com.3t.Robomongo.plist
```

To also backup the configured connections:

```suggestion
.3T
```